website/docs/concepts/event-bridge/event-bus.md

```diff
@@ -6,343 +6,125 @@ sidebar_position: 1
 
 An `EventBus` ingests and routes events throughout your application. Events can come from and be sent to any service, including AWS services or non-AWS SaaS such as Slack, Stripe, etc.
 
-## Create an EventBus
-
-Import the `EventBus` Construct and instantiate it.
+## New Event Bus
 
 ```ts
 import { EventBus } from "functionless";
 
-const bus = new EventBus(scope, "Bus");
-```
-
-## Declare the types of Events
-
-It is recommended to declare types for each of the event types flowing through an `EventBus`.
-
-First, declare an `interface` representing an Event's payload.
+new EventBus(stack, "bus");
 
-```ts
-interface UserDetails {
-  id?: string;
-  name: string;
-  age: number;
-  interests: string[];
-}
+// to name the bus, use props
+new EventBus(stack, "bus2", { eventBusName: "myBus" });
 ```
 
-Then, declare an `interface` for the Event's envelope - i.e. the shape of the data
+## Default Bus
 
-```ts
-interface UserEvent
-  extends functionless.EventBusRuleInput<
-    UserDetails,
-    // We can provide custom detail-types to match on
-    "Create" | "Update" | "Delete"
-  > {}
-```
+There is a default Event Bus in every region of an AWS account. It contains events emitted by your AWS Resources, such as when a Step Function execution completes, or when a scheduled trigger fires.
 
-## Filter Events with an EventBusRule
-
-An `EventBusRule` filters events flowing through an `EventBus` using some conditional expression.
+Functionless provides an easy way to work with the default bus.
 
 ```ts
-const onSignUp = bus.when(
-  scope,
-  "OnSignUp",
-  (event) => event["detail-type"] === "SignUp"
-);
+const defaultBus = EventBus.default();
```

Comment:
Why does it require a function call instead of just being static?

Reply:
Err, there is a parameter for the scope that I missed. will fix.